br/pkg/restore/split.go

```diff
@@ -112,12 +112,13 @@ SplitRegions:
 	for i := 0; i < SplitRetryTimes; i++ {
 		regions, errScan := PaginateScanRegion(ctx, rs.client, minKey, maxKey, ScanRegionPaginationLimit)
 		if errScan != nil {
+			if berrors.ErrPDBatchScanRegion.Equal(errScan) {
+				log.Warn("inconsistent region info get.", logutil.ShortError(errScan))
```

Comment:
include the minKey and maxKey into the log? the (short) error message according to #27426 seems to be only "batch scan region".

Reply:
OK, let's redact it then.